app/forms/Invite_by_email_form.rb

```diff
@@ -0,0 +1,33 @@
+class InviteByEmailForm
```

Comment:
On my mind, it's a bit strange, that one form validates one email and an array of emails at the same time. I'd validate it always as an array (just split the string from a form by delimiter). As a result, you will have less code because you don't need to care about the format of that string (I mean if it's one email or many)